src/Client/Aws/AbstractAwsClient.php

```diff
@@ -14,24 +15,31 @@ abstract class AbstractAwsClient implements ClientInterface
      */
     protected $client;
 
+    /**
+     * @var SerializerInterface
+     */
+    protected $serializer;
```

Comment:
just a question: why are we enforcing the serializer dependency?

Reply:
So I can use it to serialize the envelope and the message objects: https://github.com/kfzteile24/queue-bundle/pull/3/files#diff-6cb9297c08999d7da5370fe2123cb0deR118

Reply:
I would also like to use it for Sqs serialization in the future.

Reply:
ok, if you feel like the abstraction makes sense, I wan't object : )